labs/K33402/Naseebullah_Wali/Lab_1/DatailScript.js

```diff
@@ -0,0 +1,73 @@
+document.addEventListener("DOMContentLoaded", function(event) {
+    let Url_api= 'https://www.googleapis.com/books/v1/volumes?q='
+    let author,title,publisher,bookIsbn,Img_link, page,language,description;
+    let output = document.getElementById('Output-js')
+    let params = new URLSearchParams(document.location.search);
+    let UrlID = params.get("id");
+
+
+
+    fetch(Url_api+UrlID)
```

Comment:
```
`${URL_API}${urlID}`
```

https://www.w3docs.com/snippets/javascript/how-to-do-string-interpolation-in-javascript.html